dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/ContainerHolder.java

```diff
@@ -0,0 +1,47 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import com.dotmarketing.portlets.templates.design.util.PreviewTemplateUtil;
+
+import java.util.List;
+
+/**
+ * Created by freddyrodriguez on 11/13/17.
+ */
+public class ContainerHolder {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)